src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements GetFigureArea {
+    private double side1;
+    private double side2;
+
+    @Override
+    public String toString() {
+        return "Figure - Rectangle: side1 " + this.side1
+                + " ,side2 " + this.side2
+                + " ,area " + this.getArea() + " color "
+                + this.color;

```

Comment:
The `color` field is accessed directly here. Consider using a getter method if encapsulation is a concern, especially if the `color` field is private in the `Figure` class.